src/Transport/NServiceBus.AzureServiceBus.csproj

```diff
@@ -18,8 +18,8 @@
     <PackageReference Include="Fody" Version="2.*" PrivateAssets="All" />
     <PackageReference Include="Janitor.Fody" Version="1.*" PrivateAssets="All" />
     <PackageReference Include="Obsolete.Fody" Version="4.*" PrivateAssets="All" />
-    <PackageReference Include="Particular.CodeRules" Version="*" PrivateAssets="All" />
-    <PackageReference Include="Particular.Packaging" Version="*" PrivateAssets="All" />
+    <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
```

Comment:
Is this change needed?

Reply:
Yes. VS 2017 15.5.0 is shipping a new NuGet client that is failing with wildcards.

Reply:
Sigh, nuget

Reply:
We love nuget!
![image](https://user-images.githubusercontent.com/1309622/33629758-714c01de-d9c2-11e7-872a-d3898878d66e.png)